ludos/backend/src/services/user.service.ts

```diff
@@ -179,4 +180,21 @@ export class UserService {
     await this.userRepository.save(user);
     return new ChangePasswordResponseDto(true);
   }
+
+  public async getUserInfo(userId: string): Promise<GetUserInfoResponseDto> {
```

Comment:
It would be better if the user with all relations is fetched in one query since we will have additional relations. You can check [this](https://github.com/typeorm/typeorm/issues/5366)